website/addons/wiki/templates/edit.mako

```diff
@@ -39,15 +51,9 @@
            </div>
         </div>
 
-        <!-- Menu without toggle in XS size only -->
-        <div class="osf-panel visible-xs">
-            <div class="osf-panel-header"> <i class="fa fa-list"> </i>  Menu </div>
-            <div class="osf-panel-body ">
-                <%include file="wiki/templates/toc.mako"/>
-            </div>
-        </div>
```

Comment:
@caseyrollins It looks like you removed the narrow size specific panel doign the toggling. Have you tested this in small screens?